Mathlib/RingTheory/Flat/Algebra.lean

```diff
@@ -59,7 +59,7 @@ theorem isBaseChange [Algebra R S] (R' : Type w) (S' : Type t) [CommRing R'] [Co
 
 end Algebra.Flat
 
-/-- A `RingHom` is flat if `R` is flat as an `S` algebra. -/
+/-- A `RingHom` is flat if `S` is flat as an `R` algebra. -/
```

Comment:
```suggestion
/-- A ring homomorphism `f : R →+* S` is flat if `S` is flat as an `R` algebra. -/
```

It's best to include the roles of the variables, because when someone hovers over RingHom.Flat in VS Code for their ring homomorphism from A to B, or maybe from S to R, they'll see you talking about R and S without any context.